2021-11-21 14:01:44,650 INFO    MainThread:8448 [wandb_setup.py:_flush():71] setting env: {}
2021-11-21 14:01:44,650 INFO    MainThread:8448 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-21 14:01:44,650 INFO    MainThread:8448 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ruben\Downloads\VDoomLearning\src\wandb\run-20211121_140144-1xiwwsve\logs\debug.log
2021-11-21 14:01:44,650 INFO    MainThread:8448 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ruben\Downloads\VDoomLearning\src\wandb\run-20211121_140144-1xiwwsve\logs\debug-internal.log
2021-11-21 14:01:44,651 INFO    MainThread:8448 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000255C9608860>
2021-11-21 14:01:44,652 INFO    MainThread:8448 [wandb_init.py:init():390] calling init triggers
2021-11-21 14:01:44,652 INFO    MainThread:8448 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-21 14:01:44,652 INFO    MainThread:8448 [wandb_init.py:init():435] starting backend
2021-11-21 14:01:44,652 INFO    MainThread:8448 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2021-11-21 14:01:44,657 INFO    MainThread:8448 [backend.py:ensure_launched():199] starting backend process...
2021-11-21 14:01:44,755 INFO    MainThread:8448 [backend.py:ensure_launched():205] started backend process with pid: 8552
2021-11-21 14:01:44,756 INFO    MainThread:8448 [wandb_init.py:init():444] backend started and connected
2021-11-21 14:01:44,883 INFO    MainThread:8448 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-21 14:01:44,885 INFO    MainThread:8448 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-21 14:01:44,885 INFO    MainThread:8448 [wandb_init.py:init():503] updated telemetry
2021-11-21 14:01:44,939 INFO    MainThread:8448 [wandb_init.py:init():533] communicating current version
2021-11-21 14:01:47,971 INFO    MainThread:8448 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-21 14:01:47,971 INFO    MainThread:8448 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-21 14:01:48,913 INFO    MainThread:8448 [wandb_init.py:init():576] starting run threads in backend
2021-11-21 14:01:49,126 INFO    MainThread:8448 [wandb_run.py:_console_start():1693] atexit reg
2021-11-21 14:01:49,127 INFO    MainThread:8448 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-21 14:01:49,128 INFO    MainThread:8448 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-21 14:01:49,130 INFO    MainThread:8448 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-21 14:01:49,130 INFO    MainThread:8448 [wandb_init.py:init():603] run started, returning control to user process
2021-11-21 14:01:49,131 INFO    MainThread:8448 [wandb_run.py:_config_callback():911] config_cb None None {}
2021-11-21 15:05:05,650 INFO    MainThread:8448 [wandb_run.py:finish():1347] finishing run teccr/VDoomLearning/1xiwwsve
2021-11-21 15:05:05,859 INFO    MainThread:8448 [jupyter.py:save_history():429] not saving jupyter history
2021-11-21 15:05:05,859 INFO    MainThread:8448 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-21 15:05:05,859 INFO    MainThread:8448 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-21 15:05:05,859 INFO    MainThread:8448 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-21 15:05:05,859 INFO    MainThread:8448 [wandb_run.py:_restore():1635] restore
2021-11-21 15:05:06,217 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 1111
}

2021-11-21 15:05:06,711 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 1111
}

2021-11-21 15:05:06,825 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:06,936 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,039 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,154 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,258 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,370 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,489 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1111
  total_bytes: 8182
}

2021-11-21 15:05:07,603 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5495
  total_bytes: 8182
}

2021-11-21 15:05:07,718 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:07,832 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:07,938 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:08,054 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:08,174 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:08,525 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}

2021-11-21 15:05:09,507 INFO    MainThread:8448 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8182
  total_bytes: 8182
}
local_info {
}

2021-11-21 15:05:10,636 INFO    MainThread:8448 [wandb_run.py:_append_history():2011] rendering history
2021-11-21 15:05:10,636 INFO    MainThread:8448 [wandb_run.py:_append_summary():1966] rendering summary
2021-11-21 15:05:10,636 INFO    MainThread:8448 [wandb_run.py:_append_files():2061] logging synced files
